lib/Dist/Zilla/Plugin/Git/Init.pm

```diff
@@ -140,11 +154,15 @@ C<OPTION VALUE>.  This may be specified multiple times to add multiple entries.
 added by default.  A remote is specified as C<NAME URL>.  This may be
 specified multiple times to add multiple remotes.
 
+=item * push_url - the URL to use to push to a particular remote to add to
```

Comment:
Can you clarify here that this is not required in a remote spec -- that the get-url is used if not specified?  Some wording from the git docs could be lifted wholesale, or a reference to the docs used.  Otherwise, the implication is that 'remote' is only good for gets, and won't be used as the fallback for pushes.

Reply:
I will check, since that IS the case. Will get back to you within a few days with updated documentation, OK?

Reply:
My apologies for not getting back sooner. (I was in the hospital for a few days of those.) Added a near-quote from the git-remote man page, and a description of the use case. I hope this satisfies what you're asking for.